src/normal.cc

```diff
@@ -391,6 +393,8 @@ void view_commands(Context& context, NormalParams params)
         case 'l':
             window.scroll( std::max<ColumnCount>(1, count));
             break;
+        default:
+            throw runtime_error("key not mapped");
```

Comment:
Good idea, thanks for the patch.
I just have one quibble: if I hit a wrong key in lock view mode (`V`),
then the error message will not go away even after I press valid keys. I think it should.

Reply:
Would be good to fix yeah, but can merge as-is and fix later.